templates/embeds/card.html.twig

```diff
@@ -1,15 +1,17 @@
 {% import '@Tabler/components/buttons.html.twig' as button %}
 {% if block('box_before') is defined %}{{ block('box_before') }}{% endif %}
 
-{% set _collapsed   = collapsed is defined ? collapsed : false %}
-{% set _collapsible = collapsible is defined ? collapsible : _collapsed %}
-{% set _fullsize    = fullsize is defined ? fullsize : false %}
-{% set _footer      = use_footer|default(false) or block('box_footer') is defined %}
-{% set _boxtype     = boxtype|default(null) %}
-{% set _id          = id|default('collapse_' ~ random()) %}
-{% set _title       = block('box_title') is defined ? block('box_title') : null %}
-{% set _tools       = block('box_tools') is defined ? block('box_tools') : null %}
-{% set _header      = block('box_header') is defined ? block('box_header') : null %}
+{% set _collapsed           = collapsed is defined ? collapsed : false %}
+{% set _collapsible         = collapsible is defined ? collapsible : _collapsed %}
+{% set _collapsible_class   = 'collapse_' ~ random() %}
+{% set _fullsize            = fullsize is defined ? fullsize : false %}
+{% set _footer              = use_footer|default(false) or block('box_footer') is defined %}
+{% set _footer_collapsible  = footer_collapsible is defined ? footer_collapsible : false %}
+{% set _boxtype             = boxtype|default(null) %}
+{% set _id                  = id|default(null) %}
```

Comment:
`id` still there

Reply:
Yeah, but not filled. I know I have some automatic Javascript that fetches the ID and uses it for xyz (can't remember what it does 😁). 

It is simply a BC break that someone needs to be aware of. So it should be documented in the [migration guide](https://github.com/kevinpapst/TablerBundle/blob/main/docs/migration_guide.md) (I already added it). 

I have therefor added the first documentation files at https://github.com/kevinpapst/TablerBundle/tree/main/docs

Reply:
> Yeah, but not filled

I don't get it, how it is not filled ?
If you pass `id` parameter to the embed, works fine.

You maybe want a "default" id for your case ?
We can add that instead of `default(null)`, to like `default('card_' ~ random())`.
But I don't know if it's a good idea to auto implement id for card(s) embed.

Reply:
My JS relied on the ID and I did not pass it manually, because there is a potential unlimited number of cards. Thats why the auto ID was there in the first place. But it is okay, I can change my JS as well.

I don't even know if we should add the automatic class, now that you removed the automatic ID assignment. If we allow passing these values, it is the responsibility of the developer of the theme to take care of his requirements.

Reply:
> it is the responsibility of the developer of the theme to take care of his requirements

I agree 👍